gcasc/__init__.py

```diff
@@ -8,6 +8,7 @@
 from .utils import logger as logging
 
 from .base import Mode
+from .appearance import AppearanceConfigurer
```

Comment:
Just a minor thing here, to my understanding relative imports should be avoided :)

Reply:
Why?